RecoLocalCalo/HcalRecAlgos/interface/PulseShapeFitOOTPileupCorrection.h

```diff
@@ -130,6 +130,7 @@ class PulseShapeFitOOTPileupCorrection
     double pedSig_;
     double noise_;    
     HcalTimeSlew::BiasSetting slewFlavor_;    
+    const double overlapLimit = 1.0;
```

Comment:
I don't know if I can attached file here. Just try it. Attached is some
slides from Stephanie showing this overlapping study. On page 2, it shows
the timing difference distribution, and you can see dt<1 ns (3000 pulses)
and dt<10 ns (3000+40*18 bins=3720 pulses) doesn't make any difference. And
I think for a difference above 10 ns, one can distinguish the two pulses.
How about just a 10 ns limit?

On Thu, Jan 15, 2015 at 1:55 PM, Mark Grimes <notifications@github.com>
wrote:

> In RecoLocalCalo/HcalRecAlgos/interface/PulseShapeFitOOTPileupCorrection.h
> <https://github.com/cms-sw/cmssw/pull/7174#discussion-diff-23007511>:
>
> > @@ -130,6 +130,7 @@ class PulseShapeFitOOTPileupCorrection
> >      double pedSig_;
> >      double noise_;
> >      HcalTimeSlew::BiasSetting slewFlavor_;
> > +    const double overlapLimit = 1.0;
>
> This is ns right? I'm not sure this is big enough, since each data point
> in the fit is 25 ns wide. Hence pulses could be up to 24.9999 ns apart and
> still be referring to the same bin on the distribution. In my opinion it
> would be better to check which bin the pulses refer to. For the short term
> though I think this code is okay although I'll make the overlapLimit user
> configurable (I can do this in another pull request).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/7174/files#r23007511>.
>